TwitchPlaysAssembly/Src/UI/ModuleCameras.cs

```diff
@@ -842,6 +847,18 @@ public IEnumerator DisableCameras()
 	#endregion
 
 	#region Private Methods
+	private IEnumerator HideNotesForModules()
+	{
+		while (true)
+		{
+			yield return null;
+			if (ModulesHidingNotes > 0)
+				SetHudVisibility(false);
+			else if (_currentBomb != null)
+				SetHudVisibility(true);
+		}
+	}
```

Comment:
The problem I have with using a setter is mainly edge cases. Like what if both TSH and DIW call for the notes to be hidden at the same time. Sure it will get hidden but if it just calls a coroutine from the mod and ends for the mod the other might not be done yet. Future mods could also come out with subtitles hidden by the notes where it needs to be hidden too so that's why I thought this would work best.

Reply:
Yeah the only problem I encountered when testing this change was if you dropped and held the bomb again then it would put the notes back over the text being displayed which the coroutine prevented. All I had to do to fix this though was prevent it from doing this on bomb hold if there are modules hiding the notes.